vendor/vaimo/webdriver-binary-downloader/src/Interfaces/PlatformAnalyserInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
```

Comment:
This file doesn't have to be committed. (as it is in `vendor/`)

Reply:
Sure :) Just needed this dirty quick-fix to get the test working on my system. I should not forget to remove it from the commits once everything is ready..